docs/user/cli/export.rst

```diff
@@ -111,8 +122,11 @@ The ``--output-format`` argument defines the encoding used for export. Currently
     map generation it is highly recommended to use GeoServer. Additionally, the resulting file from this command
     requires the use of an online browser to open in order to access online resources.
 
-The ``--output`` argument can be used to export to a file. By default, export data is written to the standard
-output stream.
+The ``--output`` argument can be used to export to a file. By default, export data is written to standard output.
+
+The ``--sort-by`` argument can be used to sort the output by one or more attributes. Note that for local exports,
+this will usually be done in-memory, so can be costly for large result sets. By default, output is sorted in
```

Comment:
Are there options here to spill to disk for the sort?

Reply:
no, we haven't implemented that AFAIK, it just sets the query sort and the data store handles it as normal.